docs/content/preview/yugabyte-platform/_index.md

```diff
@@ -10,12 +10,16 @@ menu:
     parent: yugabytedb-anywhere
     identifier: overview-yp
     weight: 10
-type: indexpage
```

Comment:
I think type needs to stay as indexpage with breadcrumbDisable true, then the layout should sort out; I think homepage only works for the root page

Reply:
This doesn't work. I committed my changes before the phase 1 went live. Everything looked fine on my localhost. But now, if I add type: indexpage, the page only shows the left-side menu. I think I need to merge the PR (without type) and then try to add type through a second PR.